dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java

```diff
@@ -889,6 +889,45 @@ public static void startTransaction()  throws DotHibernateException{
 		}
 	}
 
+	/**
+	 * This just commit does not close anything.
+	 * @throws DotHibernateException
+	 */
+	public static void onlyCommitTransaction()  throws DotHibernateException{
+
+		try{
+			// if there is nothing to close
+			if (sessionHolder.get() == null){
+				return;
+			}
+			Session session = getSession();
+
+			if (session != null) {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this expression which always evaluates to "true" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2589)